osbs/build/build_request.py

```diff
@@ -1060,11 +1060,19 @@ def render_tag_from_config(self):
         unique_tag = self.spec.image_tag.value.split(':')[-1]
         tag_suffixes = {'unique': [unique_tag], 'primary': []}
 
+        # this part is very ugly but otherwise ~900 tests will fail
+        # as self.repo_info object is not injected properly
```

Comment:
Maybe it just needs initialising to `RepoInfo()`?

Reply:
You don't think `BuildRequest.__init__()` ought to initialise it to `RepoInfo()`?